test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -81,7 +81,17 @@ var helpers = function helpers() {
             for (name in o) {
                 this.writeln(name, o.hasOwnProperty(name) ? "" : " (inherited)", ": ", o[name]);
             }
-        }
+        },
+
+        withPropertyDeleted: function withPropertyDeleted(object, propertyName, callback) {

```

Comment:
I actually don't know. According to the [spec](https://tc39.github.io/ecma262/#sec-object-initializer-runtime-semantics-propertydefinitionevaluation), the `name` property would return the property name before the colon without the name in the function definition, and we implement this correctly. I'll see if removing them causes any test failures.